<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MySQLInfoDlgClass</class>
 <widget class="QDialog" name="MySQLInfoDlgClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>484</width>
    <height>365</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MySQLInfoDlg</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QGroupBox" name="groupBox">
       <property name="title">
        <string>Connected to MySQL Server Instance</string>
       </property>
       <widget class="QWidget" name="">
        <property name="geometry">
         <rect>
          <x>12</x>
          <y>20</y>
          <width>408</width>
          <height>58</height>
         </rect>
        </property>
        <layout class="QFormLayout" name="formLayout">
         <item row="0" column="0">
          <widget class="QLabel" name="label">
           <property name="minimumSize">
            <size>
             <width>100</width>
             <height>12</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>100</width>
             <height>12</height>
            </size>
           </property>
           <property name="text">
            <string>UserName:</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QLabel" name="m_userName">
           <property name="minimumSize">
            <size>
             <width>300</width>
             <height>12</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>300</width>
             <height>12</height>
            </size>
           </property>
           <property name="text">
            <string>UserName:</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="label_2">
           <property name="minimumSize">
            <size>
             <width>100</width>
             <height>12</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>100</width>
             <height>12</height>
            </size>
           </property>
           <property name="text">
            <string>Host:</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QLabel" name="m_host">
           <property name="minimumSize">
            <size>
             <width>300</width>
             <height>16</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>300</width>
             <height>16</height>
            </size>
           </property>
           <property name="text">
            <string>Host:</string>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="label_3">
           <property name="minimumSize">
            <size>
             <width>100</width>
             <height>12</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>100</width>
             <height>12</height>
            </size>
           </property>
           <property name="text">
            <string>Port:</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QLabel" name="m_port">
           <property name="minimumSize">
            <size>
             <width>300</width>
             <height>16</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>300</width>
             <height>16</height>
            </size>
           </property>
           <property name="text">
            <string>Port:</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox_2">
       <property name="title">
        <string>Server Information</string>
       </property>
       <widget class="QWidget" name="">
        <property name="geometry">
         <rect>
          <x>11</x>
          <y>21</y>
          <width>408</width>
          <height>50</height>
         </rect>
        </property>
        <layout class="QFormLayout" name="formLayout_2">
         <item row="0" column="0">
          <widget class="QLabel" name="label_7">
           <property name="minimumSize">
            <size>
             <width>100</width>
             <height>12</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>100</width>
             <height>12</height>
            </size>
           </property>
           <property name="text">
            <string>MySQL Version:</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QLabel" name="m_serverVersion">
           <property name="minimumSize">
            <size>
             <width>300</width>
             <height>12</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>300</width>
             <height>12</height>
            </size>
           </property>
           <property name="text">
            <string>MySQL Version:</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="label_8">
           <property name="minimumSize">
            <size>
             <width>100</width>
             <height>12</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>100</width>
             <height>12</height>
            </size>
           </property>
           <property name="text">
            <string>Network Name:</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QLabel" name="m_serverNetworkName">
           <property name="minimumSize">
            <size>
             <width>300</width>
             <height>12</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>300</width>
             <height>12</height>
            </size>
           </property>
           <property name="text">
            <string>Network Name:</string>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="label_9">
           <property name="minimumSize">
            <size>
             <width>100</width>
             <height>12</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>100</width>
             <height>12</height>
            </size>
           </property>
           <property name="text">
            <string>IP:</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QLabel" name="m_serverIP">
           <property name="minimumSize">
            <size>
             <width>300</width>
             <height>12</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>300</width>
             <height>12</height>
            </size>
           </property>
           <property name="text">
            <string>IP:</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox_3">
       <property name="title">
        <string>Client Information</string>
       </property>
       <widget class="QWidget" name="">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>20</y>
          <width>408</width>
          <height>86</height>
         </rect>
        </property>
        <layout class="QFormLayout" name="formLayout_3">
         <item row="0" column="0">
          <widget class="QLabel" name="label_13">
           <property name="minimumSize">
            <size>
             <width>100</width>
             <height>12</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>100</width>
             <height>12</height>
            </size>
           </property>
           <property name="text">
            <string>Version:</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QLabel" name="m_clientVersion">
           <property name="minimumSize">
            <size>
             <width>300</width>
             <height>12</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>150</width>
             <height>12</height>
            </size>
           </property>
           <property name="text">
            <string>Version:</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="label_14">
           <property name="minimumSize">
            <size>
             <width>100</width>
             <height>12</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>100</width>
             <height>12</height>
            </size>
           </property>
           <property name="text">
            <string>Network Name:</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QLabel" name="m_clientNetworkName">
           <property name="minimumSize">
            <size>
             <width>150</width>
             <height>12</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>300</width>
             <height>12</height>
            </size>
           </property>
           <property name="text">
            <string>Network Name:</string>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="label_15">
           <property name="minimumSize">
            <size>
             <width>100</width>
             <height>12</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>100</width>
             <height>12</height>
            </size>
           </property>
           <property name="text">
            <string>IP:</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QLabel" name="m_clientIP">
           <property name="minimumSize">
            <size>
             <width>300</width>
             <height>12</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>300</width>
             <height>12</height>
            </size>
           </property>
           <property name="text">
            <string>IP:</string>
           </property>
          </widget>
         </item>
         <item row="3" column="0">
          <widget class="QLabel" name="label_16">
           <property name="minimumSize">
            <size>
             <width>100</width>
             <height>12</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>100</width>
             <height>12</height>
            </size>
           </property>
           <property name="text">
            <string>Operating System:</string>
           </property>
          </widget>
         </item>
         <item row="3" column="1">
          <widget class="QLabel" name="m_clientOS">
           <property name="minimumSize">
            <size>
             <width>300</width>
             <height>12</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>300</width>
             <height>12</height>
            </size>
           </property>
           <property name="text">
            <string>Operating System:</string>
           </property>
          </widget>
         </item>
         <item row="4" column="0">
          <widget class="QLabel" name="label_17">
           <property name="minimumSize">
            <size>
             <width>100</width>
             <height>12</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>100</width>
             <height>12</height>
            </size>
           </property>
           <property name="text">
            <string>Hardware:</string>
           </property>
          </widget>
         </item>
         <item row="4" column="1">
          <widget class="QLabel" name="m_clientHardware">
           <property name="minimumSize">
            <size>
             <width>300</width>
             <height>12</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>300</width>
             <height>12</height>
            </size>
           </property>
           <property name="text">
            <string>Hardware:</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
